src/v3/core/graph.test.js

```diff
@@ -1,35 +1,14 @@
 // @flow
 
-import {Address, Direction, Graph, edgeToString} from "./graph";
-import type {NodeAddress, EdgeAddress} from "./graph";
+import {
+  Direction,
+  EdgeAddress,
+  Graph,
+  NodeAddress,
+  edgeToString,
+} from "./graph";
 
 describe("core/graph", () => {
-  const {nodeAddress, edgeAddress} = Address;
-  describe("Address re-exports", () => {
-    it("exist", () => {
-      expect(Address).toEqual(expect.anything());
-    });
-    it("include distinct NodeAddress and EdgeAddress types", () => {
```

Comment:
(Also referenced on the first PR) - I like this test and think it should continue in some form.

Reply:
Reinstated in `graph.test.js` in #355.